macos/README.md

```diff
@@ -1,14 +1,30 @@
+# Fluent UI for macOS
 
-# Contributing
+##### The macOS UI framework for building experiences for Microsoft 365.
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+FluentUI for macOS is a native library that provides the Microsoft UI experience for the macOS platform. It contains custom controls and customizations for platform controls, all from the official Fluent design language used in Microsoft 365 products.
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+## Contents
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+- [Controls](#controls)
+- [Test app](#test-app)
+
+## Controls
+
+FluentUI for macOS includes an expanding library of controls written in Swift and supporting Objective-C. These controls implement the Fluent Design language and provide consistency across Microsoft experiences.
+
+Some of the controls available include:
+- AvatarView
+- DatePicker

```

Comment:
When we have a more fully featured control set we'll probably want to remove it, right? Should we just do that now?